doc/announcement.html

```diff
@@ -1,6 +1,6 @@
 <div class="sidebar-message">
-    <strong>
-        This package is a work in progress. It hasn't been released yet.
-        To use it anyway, <a href="https://skrub-data.org/stable/install.html">install it from source</a>.
-    </strong>
+    <strong>This package is a 🚧 work in progress ⚠️.</strong>
```

Comment:
for some reason that part gets loaded directly from the repo with javascript when the page is rendered; so we will not see it in the rendered docs until the PR is merged in main. but you can check the wording here

Reply:
@GaelVaroquaux  if you want to check the message

Reply:
I'm not sure about this one; all packages are work in progress. The explainer right after is more explicit.